File: articles/responsible-automation-and-decision-delegation/java/DelegationReadinessScore.java
public class DelegationReadinessScore {
    public static void main(String[] args) {
        double[] scores = {0.62, 0.58, 0.46, 0.52, 0.60, 0.58};
        double total = 0.0;
        for (double score : scores) total += score;
        System.out.printf("delegation_readiness_score=%.4f%n", total / scores.length);
    }
}

File: articles/responsible-automation-and-decision-delegation/go/delegation_readiness_score.go
package main

import "fmt"

func main() {
	scores := []float64{0.62, 0.58, 0.46, 0.52, 0.60, 0.58}
	total := 0.0
	for _, score := range scores {
		total += score
	}
	fmt.Printf("delegation_readiness_score=%.4f\n", total/float64(len(scores)))
}
